assets/scss/templates/_reportes.scss

```diff
@@ -81,7 +105,7 @@
 /* Section KPI */
 .container-reports {
   width: 100%;
-  padding: 60px 0;
+  padding: 60px 2%;
```

Comment:
percentage? is it fine?

Should you not use the levels?

Reply:
I saw that you have changed it by levels, but, I am still curious, is it the same? are levels not absolute and percentage relative? how are they equivalent?

Reply:
El porcentaje es relativo y los niveles son absolutos, solo tome algo proporcional. no es lo mismo. se uso porcentaje para que sea relativo al tamaño de pantalla. Pero como planteamos niveles no estaría mal utlizarlos.